src/components/dock/dock.types.ts

```diff
@@ -0,0 +1,88 @@
+export interface DockItem {
```

Comment:
@Kiarokh @LucyChyzhova I removed some things, like colors for example, from the interface (temporarily), because I wanted to make it as clean as possible to start out from, both to make things easier for myself and to make it a little easier to show and tell about my changes.

There's no problem adding them back in 😊

Reply:
(Although, do we really need to set stuff like button background colour? Do we have an existing use case for that? 🤔)

Reply:
would these examples suffice 😄 
<img width="525" alt="image" src="https://user-images.githubusercontent.com/35954987/180444957-28bf1c51-6843-4582-9845-1068b4d9b58d.png">
<img width="468" alt="image" src="https://user-images.githubusercontent.com/35954987/180445031-52f3e78c-e86a-4ef0-9b4c-302c21b93d48.png">

I just feel colors are so basic things that most consumers would probably wanna tweak. 
So I'd rather have them back. We also have an example that showcased usage of colors.

Maybe having button background color sounds a bit overkill. But icon color, not :)

Reply:
Ah, ok, yeah, that's definitely example enough! 👍 😄

Reply:
Hmm… I checked out the commit just before my first commit, and the example _ Using color properties for individual Dock items_ only seems to support `iconColor`. `selectedBackgroundColor` and `selectedTextColor` doesn't seem to be applied at all? 🤨

I think we should get a first version functional and merged, and add further support for colours and other extras once we have a working MVP.

Reply:
If we start with `iconColor`, we need to add tweaks in the CSS files as well. I need to see the code to be able to answer why you don't get colors.

However, I think we can drop `selectedBackgroundColor` and `selectedTextColor` for now. If you push, I will tweak the CSS accordingly